pkgs/tools/misc/yajsv/default.nix

```diff
@@ -1,24 +1,22 @@
 { buildGoModule, fetchFromGitHub, lib }:
 
-let version = "1.4.0";
-in buildGoModule {
+buildGoModule rec {
```

Comment:
I would prefer to leave the let binding intact. See https://nix.dev/anti-patterns/language#rec-expression